lib/runners/processor/eslint.rb

```diff
@@ -200,7 +200,7 @@ def run_analyzer(config: nil)
         Results::Success.new(guid: guid, analyzer: analyzer)
       elsif no_eslint_config?(stderr)
         trace_writer.message "Retrying with the default configuration file because no configuration files were found..."
-        FileUtils.copy_file(DEFAULT_ESLINT_CONFIG, ".eslintrc.yml")
+        FileUtils.copy_file(DEFAULT_ESLINT_CONFIG, ".eslintrc.js")
```

Comment:
[suggestion] Can you refactor it as follows?

```suggestion
        FileUtils.cp(DEFAULT_ESLINT_CONFIG, ".")
```

Reply:
If I changed this, Smoke tests take very long time, and show below errors.

> ? Pattern matching failed at [:result].keys.sort:
--- expected
+++ actual
@@ -1 +1 @@
-[:analyzer, :guid, :issue_count, :issues, :timestamp, :type]
+[:analyzer, :guid, :message, :timestamp, :type]
? Pattern matching failed at [:result][:type]:
Expected: "success"
  Actual: "failure"
? Pattern matching failed at [:result][:message]:
--- expected
+++ actual
@@ -1 +1 @@
-nil
+"The analysis failed due to an unexpected error. See the analysis log for details."
? mismatched_package_version  4821.0s
? Pattern matching failed at [:result].keys.sort:
--- expected
+++ actual
@@ -1 +1 @@
-[:analyzer, :guid, :issue_count, :issues, :timestamp, :type]
+[:analyzer, :guid, :message, :timestamp, :type]
? Pattern matching failed at [:result][:type]:
Expected: "success"
  Actual: "failure"
? Pattern matching failed at [:result][:message]:
--- expected
+++ actual
@@ -1 +1 @@
-nil
+"The analysis failed due to an unexpected error. See the analysis log for details."
? no_ignore  4849.6s

Reply:
Oh, sorry. My suggestion is wrong. 🙇🏼